config/spec/mainnet.go

```diff
@@ -129,10 +143,14 @@ func MainnetChainSpecData() *chain.SpecData {
 		BytesPerBlob:                     defaultBytesPerBlob,
 		KZGCommitmentInclusionProofDepth: defaultKZGCommitmentInclusionProofDepth,
 
-		// Berachain values.
+		// Berachain values at genesis.
 		ValidatorSetCap:      MainnetValidatorSetCap,
 		EVMInflationAddress:  common.NewExecutionAddressFromHex(MainnetEVMInflationAddress),
 		EVMInflationPerBlock: MainnetEVMInflationPerBlock,
+
+		// Deneb1 values.
+		EVMInflationAddressDeneb1:  common.NewExecutionAddressFromHex(MainnetEVMInflationAddressDeneb1),
+		EVMInflationPerBlockDeneb1: MainnetEVMInflationPerBlockDeneb1,
```

Comment:
@calbera these are not the values at genesis right?

Reply:
these are the Deneb1 values